lib/codeowners/cli/main.rb

```diff
@@ -11,15 +11,15 @@ module Codeowners
   module Cli
     # Command Line Interface used by bin/codeowners-checker.
     class Main < Base # rubocop:disable Metrics/ClassLength
-      LABEL = { missing_ref: 'Missing references', useless_pattern: 'No files matching with the pattern' }.freeze
```

Comment:
This does not seem to be used anywhere. It was introduced in https://github.com/toptal/codeowners-checker/commit/eb01b5c2596d548e573a0c3845c8a6d8df97fabc but does not seem to be used there either.

Reply:
`grep -rnA3 LABEL`:
```
lib/codeowners/cli/main.rb:14:      LABEL = { missing_ref: 'Missing references', useless_pattern: 'No files matching with the pattern' }.freeze
--
lib/codeowners/cli/main.rb:256:      LABELS = {
lib/codeowners/cli/main.rb-257-        missing_ref: 'No owner defined',
lib/codeowners/cli/main.rb-258-        useless_pattern: 'Useless patterns'
lib/codeowners/cli/main.rb-259-      }.freeze
--
lib/codeowners/cli/main.rb:265:          puts LABELS[error_type], '-' * 30, inconsistencies, '-' * 30
```
go ahead and drop it